api/controllers/UserController.js

```diff
@@ -248,10 +254,17 @@ module.exports = {
       }
     )
     var total
-
     Search.forUsers(options).fetchAll({withRelated: ['skills', 'organizations', 'memberships']})
     .tap(users => total = (users.length > 0 ? users.first().get('total') : 0))
     .then(users => users.map(u => UserPresenter.presentForList(u, res.locals.community.id)))
+    .then(list =>
```

Comment:
this is fine by me. note though, as i mentioned in slack, that the tags for a user should be the ones directly associated with them (tags_users), aka 'skills', not the ones associated with posts they've created